MapboxCoreNavigation/PassiveLocationDataSource.swift

```diff
@@ -117,7 +117,8 @@ extension PassiveLocationDataSource: CLLocationManagerDelegate {
             return
         }
         
-        let status = navigator.status(at: lastRawLocation.timestamp)
+        let timestampForStatus = Date().addingTimeInterval(TimeInterval(1.1))
```

Comment:
@mapbox/navnative, is it a safe assumption that we should always ask the navigator for the location 1.1&nbsp;seconds ahead during free-driving, but not during active turn-by-turn navigation? Would this duration vary by any factors within the navigation SDK’s control?

Reply:
I believe this look ahead time is somehow configurable by end user in Android SDK. @mapbox/navigation-android am I right?

Also I don't see why we should do it for free-drive only. It should be done for both active guidance and free drive. But it can introduce more map matching errors, because every prediction increases chances of wrong snappings - that's why it is good idea to give end user ability to tune this parameter.

Reply:
Yes, we add prediction (1.1s by default) to both free drive and active guidance and this value is configurable with runtime options.

Reply:
OK, that addresses my concern about there being a discrepancy between navigators, given the intention to reuse the navigator between PassiveLocationDataSource and RouteController at some point.

As for the default, can we explain why it’s 1.1&nbsp;seconds and not 1.0 seconds, the actual duration of each camera animation?

https://github.com/mapbox/mapbox-navigation-ios/blob/8a3aec8b152ad1cdede6ef8494300b3e2c4406f6/MapboxNavigation/NavigationMapView.swift#L372

Reply:
@LukasPaczos confirmed that the intention is to align the fast-forwarding interval with the animation interval to avoid choppiness. The animation interval is based on location updates coming in once a second from Core Location, also to avoid choppiness. I think that speaks to a more appropriate default of 1.0&nbsp;seconds, but I’d love to hear why it should be 1.1&nbsp;seconds instead. It’s very unlikely that a 1.0-second interval would result in a user-perceptible lag where 1.1&nbsp;seconds would not.

Reply:
Like I mentioned on slack, I think we can safely change the prediction on Android to 1.0 if that simplifies the implementation on iOS. We're already multiplying the animation duration by 1.1 in the Maps SDK to have smooth transitions, so it shouldn't impact the Android experience in any significant way.